openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -3326,8 +3329,9 @@ def test_feature_recording_option_precedence(self):
 
         prob.driver.add_recorder(recorder)
         prob.driver.recording_options['record_desvars'] = True
+        prob.driver.recording_options['record_outputs'] = True
         prob.driver.recording_options['includes'] = []
-        prob.driver.recording_options['excludes'] = ['y']
```

Comment:
Does removing 'y' from excludes mean we aren't really testing precedence on excludes anymore?  I took a quick look at the example, and it doesn't even mention excludes, so I guess it doesn't, but it just seems kind of odd.

Reply:
Agree that its odd, I'll put up a new issue for it.